### DISCUSSION TITLE
Is it possible to add a paragraph to an existing docx?

### DISCUSSION THREAD

**Discussion by dentdar** [dolanmiu/docx#1912] [Q&A] [answered]
Is it possible to add a paragraph to a previously created existing docx.Document?

**Comment by dolanmiu** [comment, marked as answer]
Not possible sorry. No way to read an existing docx file

**Comment by dolanmiu** [comment]
It will be possible soon after this PR is merged:

https://github.com/dolanmiu/docx/pull/1960
